src/app/index.tsx

```diff
@@ -72,14 +73,16 @@ export function App() {
           <NetworkRibbon />
           <Switch>
             <Route exact path="/" component={BuySovPage} />
+            <Route exact path="/buy-sov" component={BuySovPage} />
             <Route exact path="/trade" component={MarginTradePage} />
             <Route exact path="/swap" component={SwapPage} />
             <Route exact path="/spot" component={SpotTradingPage} />
             <Route exact path="/lend" component={LendBorrow} />
             <Route exact path="/stats" component={StatsPage} />
-            <Route exact path="/liquidity" component={LiquidityMiningPage} />
+            <Route exact path="/yield-farm" component={LiquidityMiningPage} />
```

Comment:
Renaming brokes all external links to the dapp - should have alias for liquidity or dummy page which redirects to yield farm.

Reply:
You're right, I will add a redirect in the app.